web/src/views/CalendarLinkGeneratorView.vue

```diff
@@ -189,129 +293,22 @@ export default defineComponent({
                 calendarForm.value.location
             }&st=${convertDateTimeToPureISO(
                 calendarForm.value.startTime
-            )}&et=${convertDateTimeToPureISO(calendarForm.value.endTime)}`;
-        });
+            )}&et=${convertDateTimeToPureISO(calendarForm.value.endTime)}`
```

Comment:
variables should have spaces at the start and end like this => `${ variableName }`